frappe/model/base_document.py

```diff
@@ -963,10 +963,13 @@ def get_formatted(self, fieldname, doc=None, currency=None, absolute_value=False
 			from frappe.model.meta import get_default_df
 			df = get_default_df(fieldname)
 
-		if df.fieldtype == "Currency" and not currency:
-			currency = self.get(df.get("options"))
-			if not frappe.db.exists('Currency', currency, cache=True):
-				currency = None
+		if (
+			df.fieldtype == "Currency"
+			and not currency
+			and	(currency_field := df.get("options"))
```

Comment:
```suggestion
			and (currency_field := df.get("options"))
```
Sider is complaining about this, not visible on github or their UI too. 

<img width="639" alt="Screenshot 2022-03-23 at 6 18 47 PM" src="https://user-images.githubusercontent.com/9079960/159702931-b7949b93-4626-46c3-92c4-4a8a8872b196.png">